python/pyspark/ml/torch/tests/test_distributor.py

```diff
@@ -0,0 +1,82 @@
+from pyspark.testing.utils import PySparkTestCase
+from pyspark.ml.torch.distributor import PyTorchDistributor
+from pyspark.sql import SparkSession
+
+# Q: Do you recommend that I use pytest.mark.parametrize? It doesn't seem to be used elsewhere in this code...
+class TestPyTorchDistributor(PySparkTestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.spark = SparkSession(self.sc)
+
+    def tearDown(self):
+        super().tearDown()
+
+    def test_validate_correct_inputs(self):
```

Comment:
What is the standard for testing multiple inputs on one function. I didn't find any uses of `parametrize` so is there an alternative approach? I saw that `self.subTest()` has been used in the past so started off with that.

Reply:
we don't have it for now.  It would have to be manually parameterized as you did.